lib/SYCL/SYCLModifySPIRFuncName.cpp

```diff
@@ -0,0 +1,94 @@
+//===- SYCLModifySPIRFuncName.cpp                               ---------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+//
+// Detect functions called in SYCL kernels and modify their names.  
```

Comment:
How? Why?

Reply:
I do not understand why SPIR appears everywhere here, even in the file name.
Could be `SYCLCleanUpFuncName.cpp` instead?